0138-return-to-hnt.md

```diff
@@ -0,0 +1,163 @@
+
+The distribution schedule within each subnetwork remains under the control of subnetwork governance, and is unchanged by this proposal. All subnetwork rewards including veHNT delegations are now distributed in HNT.
+
+Subnetwork governance remains in place using the existing tokens IOT and MOBILE, which will remain available to trade on the market, though emissions cease. A suggestion is made for a future HIP that could shift veIOT and veMOBILE to a new mechanism derived from veHNT, while promoting long-term alignment to subnetworks in a way similar to the current system. Alternative forms of subnetwork governance may be proposed by the community, to be decided by a future veHNT vote.
+
+This proposal does not address other functions of the subnetwork tokens, which remain unchanged until addressed in future HIPs.
+
+### Reward Payout
+
+HNT distribution between IOT and MOBILE subnetworks continues to follow their DAO Utility Scores as per [HIP-51][hip-51] and [HIP-88][hip-88]. HNT will be rewarded directly to participants via the same claim mechanism they are familiar with, rather than deposited in the subnetwork treasuries.
+
+Rewards will be distributed according to the existing subnetwork structure and remain subject to subnetwork governance. For example, the 6% of subnetwork token emissions rewarded to delegators will instead be rewarded as 6% of the HNT awarded to that subnetwork.
+
+Emission shares which are unallocated (such as Oracle rewards) or partially unallocated (potentially MOBILE Service Provider rewards) will be burned unless a future HIP specifies otherwise.
+
+### Treasuries
+
+Treasuries retain their current HNT balances. Existing IOT and MOBILE wallet balances remain unchanged, and holders can continue to hold, or can redeem for HNT at any time at the treasury rate.
+
+### Value Imbalances
+
+Due to market factors, the IOT treasury is well-funded relative to IOT network contributions, whereas a large portion of early MOBILE rewards were minted without a backing of treasury HNT.
+
+Currently, IOT trades at its treasury rate to HNT, while MOBILE trades at a speculative premium, possibly reflecting expected future value. As a result, current rewards and valuations don’t fully match the underlying HNT emissions.
+
+1.	Higher Value of IOT Emissions: IOT rewards are valued higher in HNT than the HNT emitted to the IOT network, based on Utility Score.
+2.	Current Strong Exchange Rate for IOT Tokens: IOT converts to HNT at a strong rate, though this may decline as MOBILE network revenue increases, and HNT emissions to IOT decrease.
+3.	Lower Value of MOBILE Emissions: MOBILE rewards are worth less in HNT than the HNT emitted to the MOBILE subnetwork, due to lower HNT funding of the MOBILE treasury relative to subnetwork utility.
+4.	Market Premium on MOBILE Tokens: MOBILE trades 1.4 times higher than its treasury rate.
```

Comment:
```suggestion
1. Higher Value of IOT Emissions: IOT rewards are valued higher in HNT than the HNT emitted to the IOT network, based on Utility Score.
2. Current Strong Exchange Rate for IOT Tokens: IOT converts to HNT at a strong rate, though this may decline as MOBILE network revenue increases, and HNT emissions to IOT decrease.
3. Lower Value of MOBILE Emissions: MOBILE rewards are worth less in HNT than the HNT emitted to the MOBILE subnetwork, due to lower HNT funding of the MOBILE treasury relative to subnetwork utility.
4. Market Premium on MOBILE Tokens: MOBILE trades 1.4 times higher than its treasury rate.
```